readme.md

```diff
@@ -0,0 +1,6 @@
+# Utility

```

Comment:
Correct readme use same message as in `.nuspec`
Adjust badge links

Reply:
57ad565df9c4af4bbf2bdbedef4fc7418872b95b

Reply:
Restore Badges just adjust the links in them

Reply:
3aa7d0a599c9a4c43e976e29fb475841a5cfb335

Reply:
Remove json paket

Reply:
8b262e614b970440bcf7cd897f605ec26ac484ce comment should be by paket.dependencies